translations/de.json

```diff
@@ -0,0 +1,12 @@
+{
+  "language": {
+    "de": "Deutsch"
+  },
+  "panel": {
+    "states": "Zust&auml;nde",
```

Comment:
@armills do we need HTML entities or not? I think that if it is needed, we should add them during the build step as HTML rendering is an implementation detail of this frontend, should not drive the input.

Reply:
Wow, that was quick! I guess I'll refocus on getting our strings a little more fleshed out.

I agree. The only escape sequences we should be using here are for the JSON string itself.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Using_special_characters_in_strings